index.html

```diff
@@ -5,36 +5,35 @@
         <meta http-equiv="X-UA-Compatible" content="IE=edge" />
         <meta name="author" content="Austin">
         <link rel="stylesheet" type="text/css" href="styles.css"> 
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins"/>
+
     </head>
     <body>
       <header>
-        <nav>
+          <nav>
             <div class="logo-container">
                 <a href="#">
                     <h3 id="logo">My Logo</h3>
                 </a>
             </div>
```

Comment:
As requested by the previous CR (code reviewer) please consider removing the logo out of the nav tag to make it semantically correct like that.

```
<header>
      <div class="logo-container">
        <a href="#"> My Logo </a>
      </div>
      <nav>
        <label for="check" class="hamberger-btn">
          <img id="ham" src="img/IconMenu.png" alt="" />
        </label>
      </nav>
    </header>
```